README.md

```diff
@@ -58,7 +58,7 @@ In this example, the React Hook `useSWR` accepts a `key` and a `fetch` function.
 `data` will be `undefined`. And when we get a response, it sets `data` and `error` based on the result
 of `fetch` and rerenders the component.
 
-Note that `fetch` can be any asynchronous function, so you can use your faviourite data-fetching
+Note that `fetch` can be any asynchronous function, so you can use your favourite data-fetching
```

Comment:
```suggestion
Note that `fetch` can be any asynchronous function, so you can use your favorite data-fetching
```

Reply:
ZEIT uses US English, so this should be `favorite`.

Reply:
🤦‍♂sorry @Timer 

Could you open another PR? Thanks!